.chloggen/ottl-merge-maps.yaml~merged

```diff
@@ -0,0 +1,32 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+<<<<<<< HEAD
+<<<<<<< HEAD:.chloggen/ottl-merge-maps.yaml
+=======
+>>>>>>> 3087cc9a0e8e732303bf78d14c668e063d42e6df
```

Comment:
Please resolve the conflicts

Reply:
Oh this is actually some merge artifact and shouldn't be included in the PR

Reply:
it's still there :)

Reply:
ok, ok, NOW its gone (I think)

Reply:
It's gone but there are other unrelated changelog item added after rebasing :)